infra/build/functions/update_build_status.py

```diff
@@ -233,19 +243,58 @@ def upload_log(build_id):
 # pylint: disable=no-member
 def update_status(event, context):
   """Entry point for cloud function to update build statuses and badges."""
-  del event, context  #unused
+  del context
+
+  if 'data' in event:
+    status_type = base64.b64decode(event['data']).decode()
+  else:
+    raise RuntimeError('No data')
```

Comment:
No build status data.

Reply:
This isn't really "builds status" data, as it's referring to the generic "data" that the pub/sub trigger gives us.